src/pages/PrintCards.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useRef } from "react";
+import { useCookies } from "react-cookie";
+import { withRouter, RouteComponentProps } from "react-router-dom";
+import ReactToPrint from "react-to-print";
+import { useReactToPrint } from "react-to-print";
+import { Button } from "antd";
+import Header from "../components/Header";
+import Dimension from "../components/PrintDimension";
+import "../styles/PrintCards.css";
+import "../styles/Footer.css";
+import { API_DOMAIN } from "../config";
+
+import charts from "../dummyData/charts";
+
+const chartID = "";
```

Comment:
Is this constant necessary? It's only used in line 67 but doesn't do anything (yet at least)

Reply:
Yeah i don't think its need in this right now